### DISCUSSION TITLE
Training from classical FF

### DISCUSSION THREAD

**Discussion by JKnrd** [ACEsuit/mace#378] [Q&A]
Dear community,

I'm currently working on training a ML-FF using MACE, based on data sampled from classical FFs such as GAFF or OPLS. The sampled data comprises configurations mimicking reactions, biased within Molecular Dynamics Simulations.

However, I encountered difficulties due to the absence of isolated atom energies. This absence leads to incorrect stress calculations and ultimately results in the expansion of my simulations, a topic previously discussed (https://github.com/ACEsuit/mace/discussions/317).

In an attempt to address this issue, I experimented with setting the isolated atom energies to 0 with `E0s="isolated"`, as they are in classical FFs. Unfortunately, this approach did not resolve the problem.

I'm reaching out to you for suggestions on how to tackle this problem. Any insights or advice would be greatly appreciated.

Best JKnrd

**Comment by ilyes319** [comment]
Hey,

I think you should just ignore the energies and pass `--E0s="average"`. Make sure to then put zero loss weight to the energy, it should do it automatically but just for safety.

Overall, may I ask why you would want to train on classical FFs data? It usually not advised because there is little to be gained.
